src/input/element-input.js

```diff
@@ -614,6 +659,8 @@ pc.extend(pc, function () {
                 _y = sh * (_y - (cameraTop)) / cameraHeight;
 
                 // 3D screen
+                // TODO Should we also apply hit padding here? I wasn't sure whether UI
+                //      features such as buttons are intended to be used in 3D or not.
```

Comment:
@daredevildave @vkalpias be good to get your feedback on this

Reply:
I kinda feel like everything that applies to 2D elements should apply to 3D elements as well.